### ISSUE TITLE
Add the play count of replays on user profile page

### ISSUE THREAD

**Issue by nekodex** [ppy/osu-web#2180]
would be a nice stat to have visible

**Comment by Toybickler** [comment]
Something like this for Top ranks and 1st Place Ranks would be interesting. Can be only visible to the user too. https://i.imgur.com/jtGBG1B.png

**Comment by nekodex** [comment]
Any reason to not just have the view count publicly visible?

**Comment by Toybickler** [comment]
Not really, just considering the possibility of groups of people spamming replays to inflate numbers

**Comment by nanaya** [comment]
The number has been added both on the profile and score page.
